### PR TITLE
chore: Fix some code formats

### PR DESCRIPTION
**Pull request by Gallardot**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

ref:[CodeReviewComment](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). Fix some code formats. IDE will not remind.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by SkyeYoung** [comment]
Shouldn't code format be handled by lint or format tools? 🤔

File: cmd/cmd.go [hunk 1]
```diff
@@ -12,6 +12,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```
Comment (tao12345666333) [inline cmd/cmd.go:15]: Thank you for your contribution, I checked golang's documentation and haven't found any instructions about having to have a blank line after the license header file. 

Can you explain the need for this modification?

**Comment by tao12345666333** [comment]
Thank you for your contribution, I checked golang's documentation and haven't found any instructions about having to have a blank line after the license header file.

Can you explain the need for this modification?

**Comment by tokers** [comment]
> Shouldn't code format be handled by lint or format tools? 🤔

Yes, Code format should be handled by the gofmt tool.

**Comment by tao12345666333** [comment]
In fact, the code in the current repo has passed the gofmt check

**Comment by tokers** [comment]
> In fact, the code in the current repo has passed the gofmt check

@Gallardot Could you revert these parts?

**Review by tokers — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]
Thanks!

**Comment by Gallardot** [comment]
> Thank you for your contribution, I checked golang's documentation and haven't found any instructions about having to have a blank line after the license header file.
> 
> Can you explain the need for this modification?


It's just that some files have a newline after the license header and some don't. Just to keep the code in a consistent format.


> Shouldn't code format be handled by lint or format tools? 🤔

Agree, maybe current tools don't cover this detail. Maybe we can make some adjustments back there.

**Comment by tao12345666333** [comment]
> It's just that some files have a newline after the license header and some don't. Just to keep the code in a consistent format.

If we are modifying a feature in a file, I think it is acceptable to make adjustments to it in passing. 
But it would not be reasonable to make a centralized change to it specifically in a PR.

**Pull request merged**